bootstrap

```diff
@@ -112,23 +112,7 @@ while (true) {
   $uri = $event['path'];
 
   if (array_key_exists('multiValueQueryStringParameters', $event) && $event['multiValueQueryStringParameters']) {
-    $first = TRUE;
-    foreach ($event['multiValueQueryStringParameters'] as $name => $values) {
-      foreach ($values as $value) {
-        if ($first) {
-          $uri .= "?";
-          $first = FALSE;
-        } else {
-          $uri .= "&";
-        }
-
-        $uri .= $name;
-
-        if ($value != '') {
-          $uri .= '=' . $value;
-        }
-      }
-    }
+    $uri .= "?" . http_build_query($event['multiValueQueryStringParameters']);
```

Comment:
I don't think this really does the right thing. For example, `multiValueQueryStringParameters` might have the following value:

```JSON
    "multiValueQueryStringParameters":{
      "name":[
        "me"
      ],
      "multivalueName":[
        "you",
        "me"
      ]
    }
```

This corresponds to a url querystring like `?name=me&multivalueName=you&multivalueName=me`

If you fed the same data into `http_build_query`, it would generate the following querystring: `?name[0]=me&multivalueName[0]=you&multivalueName[1]=me`. (The square brackets `[]` would actually be url-encoded, but I left them unencoded to make it easier to understand here.)

Can you double check my thinking?

Reply:
@txase php requires the square brackets to exist for multiple value query string parameters.

if php parses the query string `?name=me&multivalueName=you&multivalueName=me` you'll end up with:
```
array(2) { ["multivalueName"]=> string(2) "me" ["name"]=> string(2) "me" } 
```

Reply:
It may be that PHP requires square brackets to exist for multiple value query string parameters, but this layer proxies parameters as they were received by API Gateway from the client. If they aren't in the correct format for PHP to understand them, then the client needs to be fixed to send them correctly.

I don't want to modify requests as they are received and then proxied to and from the PHP webserver.